### ISSUE TITLE
Feature Request: Add option to "uninstrument" from the io server

### ISSUE THREAD

**Issue by Planeshifter** [socketio/socket.io-admin-ui#63]
Is there a way to "uninstrument" the socket.io server instance to avoid any kind of performance overhead after one has disconnected from the admin dashboard?

If not, it would be nice if there was an `uninstrument` method exported from the package that would allow to unregister all the event handlers that the package is attaching to the `io` instance.

**Comment by darrachequesne** [comment]
No, it's not possible currently, but that sounds like an interesting feature :+1:

**Comment by Planeshifter** [comment]
Thanks for the quick response and confirmation that it's currently not possible!
